package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/ts-tinkerpop.js",
   "scripts": {
     "test": "make test",
-    "install": "make install-maven"
+    "install": "make install-maven",
+    "postpublish": "./devbin/update-definitely-typed.sh"
```

Comment:
If you pre-publish, then when the postpublish script runs it should be a no-op (assuming you bumped the version number proactively).

The main reason I want it as a postpublish script is just so there is an automatic reminder to do that step. If the step is already done, the reminder is unnecessary, but doesn't hurt anything.